index.js

```diff
@@ -1 +1 @@
-module.exports = require('./lib/action-types')
+module.exports = require('./lib/action-types');
```

Comment:
don't worry about adding semicolons.  I use them only where required in my projects.

Reply:
actually my IDE add them on save. It it okay to let them ?